package.json

```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "cldr-data": ">=25",
     "globalize": "^1.1.0-rc.5 <1.3.0",
-    "webpack": "^2.5.1"
+    "webpack": "^3.0.0"
```

Comment:
no it's not (therefore the bump). It was added in rc.2
As you wish. I thought the trend was to have master being the latest and greatest and patches backported to "old" branches ?